Add lighttpd logs (#7719)
DataDog/integrations-core: lighttpd/README.md

<!-- xxz tab xxx -->
<!-- xxz tabs xxx -->

#### Log collection

> 1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:

Reviewer: ```suggestion
1. Collecting logs is disabled by default in the Datadog Agent, so to enable it, open `datadog.yaml` and update the `logs_enabled` setting:
```
